### PR TITLE
[ jiefeng ] add ButtonWithColor component. it can customize the text …

### PR DESCRIPTION
**Pull request by liujiefengfeng**
add ButtonWithColor component. it can customize the text only now.

### CODE REVIEW

File: src/components/index.ts [hunk 1]
```diff
@@ -1 +1,6 @@
-export { default as Logo } from './Logo/Logo';
\ No newline at end of file
+import Logo from './Logo/Logo';
+import ButtonWithColor from './ButtonWithColor/ButtonWithColor';
+
+export {
+    Logo, ButtonWithColor
```
Comment (JellyLu) [inline src/components/index.ts]:
```
export {
Logo,
ButtonWithColor,
}
```

这样我们每个人加一行，就不容易冲突了

File: src/components/ButtonWithColor/ButtonWithColor.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+import './ButtonWithColor.css';
+
+interface ButtonWithColorProps {
+    color?: string;
```
Comment (JellyLu) [inline src/components/ButtonWithColor/ButtonWithColor.tsx]: we don't use this props?

File: src/components/index.ts (hunk 1 above)
Comment (benben198805) [reply src/components/index.ts]: 我已经感受到。

Comment (liujiefengfeng) [reply src/components/index.ts]: 好哒～～sorry for the conflict.

File: src/components/ButtonWithColor/ButtonWithColor.tsx (hunk 2 above)
Comment (liujiefengfeng) [reply src/components/ButtonWithColor/ButtonWithColor.tsx]: will use it, but it is not ready, this button  is can only customize the context now.

**Pull request merged**